_shared/project/tox.ini

```diff
@@ -29,11 +29,13 @@ setenv =
     dev: SENTRY_ENVIRONMENT = {env:SENTRY_ENVIRONMENT:dev}
     dev: NEW_RELIC_APP_NAME = {env:NEW_RELIC_APP_NAME:{{ cookiecutter.slug }}}
     dev: NEW_RELIC_ENVIRONMENT = {env:NEW_RELIC_ENVIRONMENT:dev}
+    tests: PYTEST_PLUGINS = {{ pytest_plugins("unit")|separator(",") }}
+    functests: PYTEST_PLUGINS = {{ pytest_plugins("functional")|separator(",") }}
 {% if cookiecutter.get("postgres") == "yes" %}
     SQLALCHEMY_SILENCE_UBER_WARNING=1
     dev: DATABASE_URL = {env:DATABASE_URL:postgresql://postgres@localhost:{{ cookiecutter['__postgres_port'] }}/postgres}
-    tests: TEST_DATABASE_URL = {env:TEST_DATABASE_URL:postgresql://postgres@localhost:{{ cookiecutter['__postgres_port'] }}/{{ cookiecutter.package_name }}_tests}
-    functests: TEST_DATABASE_URL = {env:TEST_DATABASE_URL:postgresql://postgres@localhost:{{ cookiecutter['__postgres_port'] }}/{{ cookiecutter.package_name }}_functests}
+    tests: DATABASE_URL = {env:DATABASE_URL:postgresql://postgres@localhost:{{ cookiecutter['__postgres_port'] }}/{{ cookiecutter.package_name }}_tests}
+    functests: DATABASE_URL = {env:DATABASE_URL:postgresql://postgres@localhost:{{ cookiecutter['__postgres_port'] }}/{{ cookiecutter.package_name }}_functests}
```

Comment:
Having the envvar be called `DATABASE_URL` in dev and prod but `TEST_DATABASE_URL` in the tests was a pain because the code when being run by the tests would try to read `DATABASE_URL` and crash. Keeping the envvar name the same just makes things simpler